src/components/policy/policy_regular/src/policy_table/types.cc

```diff
@@ -692,6 +694,8 @@ void ModuleConfig::SafeCopyFrom(const ModuleConfig& from) {
   notifications_per_minute_by_priority =
       from.notifications_per_minute_by_priority;
 
+  lock_screen_dismissal_enabled = from.lock_screen_dismissal_enabled;
```

Comment:
Missing from all other functions.

Reply:
@jacobkeeler Thanks. Fixed in 99cf2e13b8511db9c20bc0feebcca0d1610e075f

Reply:
Still missing from `is_valid` and `SetPolicyTableType`.

Reply:
@jacobkeeler Thanks. Fixed in d2956eaa741169661ae970bf4f379b86cd52bc61